src/entry.js

```diff
@@ -2,6 +2,7 @@ import Duration from 'duration'
 import shortid from 'shortid'
 import moment from 'moment'
 import parser from './parser'
+import conf from './config'
 
 export const hashPattern = /(#\w+[\w-]*)/g
 export const version = 4
```

Comment:
I think we do actually need an upgrader for this change... for user that is currently logged in, old entries will not have the user's ID associated with it. Currently the only way to handle this is to run `tick login` again.... which is not necessarily a problem. Although perhaps that is the better method? Thoughts?